packages/alert/src/Alert.tsx

```diff
@@ -69,9 +69,34 @@ export default function Alert({
   variant = 'success',
   ...other
 }: AlertProps): React.ReactElement<AlertProps> {
+  const [show, setShow] = useState(true)
   const Icon = icons[variant]
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout
+    if (type === 'toast') {
+      timer = setTimeout(hideAlert, 5 * 1000)
```

Comment:
Let's make sure there are no magic numbers here, although I think the delay should come as a prop. `5000` could be the default value. e.g:

```
const delayInMiliseconds = props.delay || 5000
```